cvat/apps/tf_annotation/views.py

```diff
@@ -174,6 +174,7 @@ def convert_to_cvat_format(data):
                 "group": None,
                 "occluded": False,
                 "attributes": [],
+                "annotation_type": "Auto",
```

Comment:
@Priya4607 , can we rename "annotation_type" into "source". Also I will vote for "auto", "manual" (fist letter should be lower case). In the future we can store in the field even username (or its id).

Reply:
Okay, should it be applied to both frontend and backend to maintain uniformity?

Reply:
I think so.